packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -254,8 +254,22 @@ class WebViewController {
     return _channel.invokeMethod("reload");
   }
 
-  Future<void> _updateSettings(Map<String, dynamic> update) async {
-    return _channel.invokeMethod('updateSettings', update);
+  Future<void> _updateSettings(_WebSettings update) async {
+    Map<String, dynamic> updateMap;
+    if (_settings == null) {
+      updateMap = update.toMap();
+    } else {
+      updateMap = _settings.updatesMap(update);
+    }
+    _settings = update;
+    return _channel.invokeMethod('updateSettings', updateMap);
+  }
+
+  /// Evaluates JavaScript in the context of the current page.
```

Comment:
The Dartdoc summary should be its own paragraph.
https://www.dartlang.org/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph